package.json

```diff
@@ -33,7 +33,7 @@
     "vinyl": "^2.1.0"
   },
   "dependencies": {
-    "event-stream": "^3.1.7",
+    "event-stream": "^3.3.5",
```

Comment:
This should be 3.3.4
> If you are a developer update your event-stream dependency to event-stream@3.3.4

https://github.com/dominictarr/event-stream/issues/116#issue-382854428
https://snyk.io/blog/malicious-code-found-in-npm-package-event-stream

Reply:
Fixed. Also pinned the other dep.

Reply:
Let's hope @mikehazell can see this PR and merge it soon...